blog/2024-06-04-how-to-choose-an-embedding-model/index.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: Step-by-Step Guide to Choosing the Best Embedding Model for Your Application
+slug: how-to-choose-an-embedding-model
+description: 'How to select an embedding model for your search and retrieval-augmented generation system.'
+authors: [leonie, zain, jp]
+date: 2024-06-04
+tags: ['how-to', 'concepts']
+image: ./img/hero.png
+---
+
+![How to select an embedding model](./img/hero.png)
+
+
+[Vector embeddings](/blog/vector-embeddings-explained) power modern search and Retrieval-Augmented Generation (RAG) applications. Embeddings capture the semantic meaning of data objects (e.g., text) and represent them in an array of numbers. In today’s Generative AI applications, these vector embeddings are typically generated with so-called embedding models.
+
+![embedding model.png](./img/embedding_model.png)
+
+As a [vector database](https://weaviate.io/blog/what-is-a-vector-database) that stores vector embeddings and retrieves data objects based on vector search, Weaviate has many [integrations with various model providers and their wide variety of embedding models](https://weaviate.io/developers/weaviate/modules/retriever-vectorizer-modules). 
```

Comment:
Maybe `https://weaviate.io/developers/weaviate/model-providers` instead of `https://weaviate.io/developers/weaviate/modules/retriever-vectorizer-modules`